runconfig/config.go

```diff
@@ -59,6 +60,10 @@ type Command struct {
 	parts []string
 }
 
+func (e *Command) ToString() string {
```

Comment:
I actually did have it called String() at first but then it broke things because when we serialized the runconfig stuff into json it would my new String() instead of the default one and the output in things like "inspect" changed. If I remember correctly, I think it dropped the [ ]'s or maybe the {}'s.  So I had to call it something else.